src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+import core.basesyntax.model.Figure;
+import core.basesyntax.suppliers.FigureSupplier;
+import java.util.Random;
+
+public class Main {
+
+    private static FigureSupplier figureSupplier = new FigureSupplier();
+
+    public static void main(String[] args) {
+        Figure[] figures = new Figure[new Random().nextInt(6)];
+
+        for (int i = 0; i < figures.length; i++) {
+            figures[i] = figureSupplier.getRandomFigure();
+        }
```

Comment:
```suggestion
        for (int i = 0; i < figures.length; i++) {
            figures[i] = i < figures.length / 2
                              ? figureSupplier.getRandomFigure()
                              : figureSupplier.getDefaultFigure();
        }
```
lets add method to get default figure

Reply:
I understand that I need to create a method getDefaultFigure() , but I don't understand what it should return?